src/views/v2/emails/template-parts/body/order/purchaser-details.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Event Tickets Emails: Order Purchaser Details
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/emails/template-parts/body/order/purchaser-details.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link https://evnt.is/tickets-emails-tpl Help article for Tickets Emails template files.
+ *
+ * @version TBD
+ *
+ * @since TBD
+ *
+ * @var Tribe_Template  $this  Current template object.
+ * @var Module           $provider              [Global] The tickets provider instance.
+ * @var string           $provider_id           [Global] The tickets provider class name.
+ * @var \WP_Post         $order                 [Global] The order object.
+ * @var int              $order_id              [Global] The order ID.
+ * @var bool             $is_tec_active         [Global] Whether `The Events Calendar` is active or not.
+ */
+
+// @todo @codingmusician @juanfra Replace hardcoded data with dynamic data.
+
+?>
+<tr>
+	<td>
+		<table>
+			<tr>
+				<th style="font-size:16px;font-weight:700;line-height:23px;text-align:left" align="left">
+					Order #123
+				</th>
+				<th style="font-size:16px;font-weight:700;line-height:23px;text-align:right" align="right">
+					David Hickox
+				</th>
+			</tr>
+			<tr>
+				<td style="font-size:14px;font-weight:400;line-height:23px;text-align:left" align="left">
```

Comment:
We also need to atomize these tiny chunks into smaller templates, that way, it's easier to customize tiny bits and maintain the whole structure. At the same time, for consistency with the other templates we have in place for the plugin.

- order/purchaser-details/order-number
- order/purchaser-details/purchaser-name
- order/purchaser-details/order-date
- order/purchaser-details/purchaser-email